src/pages/stacks-blockchain/stacking.md

```diff
@@ -0,0 +1,74 @@
+
+If you would like to implement this flow in your own wallet, exchange, or any other application, please have a look at this tutorial:
+
+[@page-reference | inline]
+| /stacks-blockchain/integrate-stacking
+
+## PoX mining
+
+PoX mining is a modification of Proof-of-Burn (PoB) mining, where instead of destroying the committed Bitcoin, it is transferred to eligible Stacks (STX) holders that participate in the Stacking protocol.
+
+-> A PoX miner can only receive newly-minted Stacks (STX) tokens when they transfer bitcoins to eligible owners of Stacks tokens
+
+Miners have to run a software (mining client, aka "miner") on their machines to participate in the PoX mechanism. The mining client implements the PoX mechanism, which ensures proper handling and incentives through four key phases:
+
+- Registration: Miners register for a future election by sending consensus data to the network
+- Commitment: Registered miners transfer bitcoins to participate in the election
+- Election: A verifiable random function chooses one miner to write a new block on the Stacks blockchain
+- Assembly: The elected miner writes the new block and collects rewards in form of new Stacks tokens
+- Distribution: Committed bitcoins are sent to a set eligible Stacks tokens holders
+
+[@page-reference | inline]
+| /mining
+
+## Token holder eligibility
+
+Stacks token holders do not automatically receive Stacking rewards. Instead, they must:
+
+- Commit to participation before a reward cycle begins
+- Hold ~94.000 Stacks tokens
+- Lock up Stacks tokens for a specified period
+- Set a Bitcoin address to receive rewards
+
+Token holders will have to use software like apps, exchanges, or wallets that support participation in Stacking.
+
+## Stacking consensus algorithm
+
+Stacking is a built-in capability of PoX and is realized through a set of actions on the Stacks 2.0 network. The full implementation details can be found in [SIP-007](https://github.com/blockstack/stacks-blockchain/blob/develop/sip/sip-007-stacking-consensus.md). Below is a summary of the most relevant actions of the algorithm.
+
+- Progression in Stacking consensus happens over reward cycles (with a fixed length). In each reward cycle, a set of Bitcoin addresses are iterated over
+- A reward cycle consists of two phases: prepare and reward
+- During the prepare phase, miners decide on an anchor block and a reward set. Mining any descendant forks of the anchor block requires transferring mining funds to the appropriate reward addresses. The reward set is the set of Bitcoin addresses which will receive funds in the reward cycle
+- Miners register as leader candidates for a future election by sending a key transaction that both burns cryptocurrency (proof-of-burn) and spends energy (proof-of-work). The transaction also registers the leader's preferred chain tip (must be a descendant of the anchor block) and commitment of funds to 5 addresses from the reward set
```

Comment:
Proof-of-work is no longer used, and the number of commitment addresses is now 2 (rather than 5).

Reply:
thanks, updated!